plugin/src/main/groovy/com/novoda/gradle/command/MonkeySpec.groovy

```diff
@@ -0,0 +1,37 @@
+package com.novoda.gradle.command;
+
+final class MonkeySpec {
+
+    static final int EVENTS_DEFAULT = 10000
+
+    def events
+    def seed
+    def categories = []
+
+    void events(events) {
+        this.events = events
+    }
+
+    void seed(seed) {
```

Comment:
Sorry it didn't work. It looks for the method `seed` and errors. This magically works with the fields in Extension though.